test/server/options_impl_test.cc

```diff
@@ -41,6 +41,8 @@ class OptionsImplTest : public testing::Test {
     options->parseComponentLogLevels(component_log_levels);
     return options->componentLogLevels();
   }
+
+  uint32_t count(const std::unique_ptr<OptionsImpl>& options) { return options->count(); }
```

Comment:
Sorry just noticed this, what's the reasoning for having this helper?

Reply:
`count() `was not exposed in the Options interface because it is not needed and is private. So added this helper in the friend class for using in it in tests. Is there any other way?

Reply:
Personally I would just make count() public on OptionsImpl (not Options) but it's not a big deal either way.

Reply:
Ah ok. Let us merge this if it is not a big deal, i will clean it up when I get opportunity?

Reply:
OK sgtm